Makefile

```diff
@@ -78,19 +79,19 @@ worker:
 
 install:
 	# GOPATH/bin must be on your PATH to access these binaries:
-	GO15VENDOREXPERIMENT=1 go install -ldflags "$(LD_FLAGS)" -gcflags "all=-trimpath=$GOPATH" ./src/server/cmd/pachctl
+	GO15VENDOREXPERIMENT=1 go install -ldflags "$(LD_FLAGS)" -gcflags "all=-trimpath=${PWD}" ./src/server/cmd/pachctl
```

Comment:
should this be `-gcflags "$(GC_FLAGS)"` as well?

Reply:
Yup, just fixed it.